lib/cisco_node_utils/interface.rb

```diff
@@ -74,8 +74,6 @@ def access_vlan
 
     def access_vlan=(vlan)
       config_set('interface', 'access_vlan', @name, vlan)
-    rescue Cisco::CliError => e
-      raise "[#{@name}] '#{e.command}' : #{e.clierror}"
```

Comment:
IMHO, it's annoying to have this pattern copy-and-pasted all over the place, and I don't like the approach of "catch one type of error and throw a new type of error with less detail" (we're discarding `e.previous`, for one). If anything, we should prefer to enhance the CliError class itself to provide the necessary context.